docs/spl-docs/evaluation-functions/trig-hyperbolic-functions.md

```diff
@@ -0,0 +1,189 @@
+# Trigonometric and Hyperbolic functions
+
+The following list contains the functions that you can use to calculate trigonometric and hyperbolic values.
+
+## acos(x)
+
+This function computes the arc cosine of `x`, in the interval `[0,pi]` radians.
+
+### Example
+
+This example returns `1.5707963267948966`.
+
+```
+... | eval n=acos(0)
+```
+
+The following example calculates 180 divided by pi and multiplies the result by the arc cosine of 0. This example returns `90.0000000003`.
+
+```
+... | eval degrees=acos(0)*180/pi()
+```
```

Comment:
You already have the `acos(0)` example, and a lot of your comment on line 17 is obvious from the code. I'd suggest either removing this example or changing the description to something more useful, such as:
```
... converts `acos(0)` from radians to degrees
```

Similar comment for your `asin` example